Header header

float64 roll
float64 pitch
float64 yaw
float64 roll_rate
float64 pitch_rate
float64 yaw_rate

float64 x
float64 y
float64 z
float64 x_rate
float64 y_rate
float64 z_rate